### DISCUSSION TITLE
Collapsible Functionality for Default Table View

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#5856] [Ideas]
Is it possible to use the collapse functionality when using the default table view (not the column layout like split or stack)? If not, this would be super helpful.

**Comment by danharrin** [comment]
The reason why this isn't supported in tables and why it won't be supported is because it's no longer an HTML table if there's a collapsible section... the layout isn't flexible enough to implement something like that

**Reply by intrepidws** [reply]
Hmm... I feel like there's a way that you could keep it as an HTML table and still accomplish this. Couldn't you just have an additional (hidden by default) `<tr>` section for every result? Something like how [this works](https://mdbootstrap.com/docs/standard/extended/responsive-table/).

**Reply by intrepidws** [reply]
Here is [another example from CSS tricks](https://css-tricks.com/table-with-expando-rows/) as well.

**Reply by danharrin** [reply]
I feel like I don't appreciate the complexity of this inside the codebase... would definitely make maintainability worse when we already have this feature available with column layouts. What's the reason why you don't like the column layout components?

**Reply by intrepidws** [reply]
For some instances like this one, I prefer having actual column headers that are sortable via click. It's easier to scan quickly with the eye IMO.
